cypress/integration/home.test.ts

```diff
@@ -0,0 +1,355 @@
+import {
+  ALGOLIA_RESPONSE_5_RESULTS,
+  ALGOLIA_RESPONSE_3_RESULTS,
+  ALGOLIA_RESPONSE_0_RESULTS,
+} from "../support/constants";
+
+describe("Home page", () => {
+  it("should allow a user to search and find 5 auto-complete results", () => {
+    cy.visit("/");
+
+    cy.intercept("**/queries?x-algolia-agent=**", {
```

Comment:
Consider prefixing with algolia.net instead of using `x-algolia-agent` (e.g. `"**algolia.net**/queries**"`). I'm worried that part would be likely to change.